dotcom-rendering/src/components/SignInGate/types.ts

```diff
@@ -118,5 +118,5 @@ export type SignInGatePropsAuxia = {
 	checkoutCompleteCookieData?: CheckoutCompleteCookieData;
 	personaliseSignInGateAfterCheckoutSwitch?: boolean;
 	userTreatment: AuxiaAPIResponseDataUserTreatment;
-	logTreatmentInteractionCall: () => Promise<void>;
+	logTreatmentInteractionCall: (actionName: string) => Promise<void>;
```

Comment:
since Auxia have given us the list of actions, how about instead of a string we define something like:
`type AuxiaInteractionEvent = 'VIEWED' | 'CLICKED' | 'DISMISSED';`

Reply:
Oh yeah. Good idea!

Reply:
I used `AuxiaInteractionActionName`, since it's an action name (although used to describe events).